### ISSUE TITLE
Batch normalization の勾配の確認

### ISSUE THREAD

**Issue by kojima-r** [kojima-r/MiniCTorch_Prototype#2]
batch_normalizationが有る場合にpytorchの結果と一致しなくなる
